modules/ppcp-subscription/src/SubscriptionsApiHandler.php

```diff
@@ -178,7 +178,15 @@ public function update_product( WC_Product $product ): void {
 						$data[] = (object) array(
 							'op'    => 'replace',
 							'path'  => '/description',
-							'value' => $product->get_description(),
+							'value' => substr(
+								strip_shortcodes(
+									wp_strip_all_tags(
+										$product->get_description()
+									)
+								),
+								0,
+								127
```

Comment:
Would be great to reuse https://github.com/woocommerce/woocommerce-paypal-payments/blob/d3a02e79c52c71bac04375ba1d93ec9858f70e3e/modules/ppcp-api-client/src/Factory/ItemFactory.php#L221-L224

e.g. extract into trait or static helper, ...